src/lib.rs

```diff
@@ -17,7 +17,7 @@
 #![feature(core_intrinsics, core_slice_ext, core_str_ext)]
 #![allow(improper_ctypes)]
 #![feature(const_fn)]
-#![deny(missing_docs)]
+// #![deny(missing_docs)]
```

Comment:
Yes, please add `#[allow(missing_docs)]` and a `TODO(jamwaffles)` to clean it up, although obviously I wouldn't force you to clean it, merely to track who placed it there